easybuild/easyconfigs/s/SIRIUS/SIRIUS-7.2.1-CrayGNU-20.11-cuda.eb

```diff
@@ -0,0 +1,38 @@
+# created by Anton Kozhevnikov (CSCS)
+easyblock = 'CMakeMake'
+
+name = 'SIRIUS'
+version = '7.2.1'
+versionsuffix = '-cuda'
+
+homepage = 'https://electronic-structure.github.io/SIRIUS/'
+description = "Domain specific library for electronic structure calculations."
+
+toolchain = {'name': 'CrayGNU', 'version': '20.11'}
+toolchainopts = {'opt': True, 'usempi': True, 'pic': True, 'verbose': False, 'openmp': True}
+
+source_urls = ['https://github.com/electronic-structure/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [
+    ('CMake', '3.14.5', '', True),
+    ('cudatoolkit', EXTERNAL_MODULE),
+    ('intel/19.1.1.217', EXTERNAL_MODULE)
```

Comment:
```suggestion
    ('intel', EXTERNAL_MODULE)
```

Reply:
If you are not bound to a specific Intel version, you can skip it and the default Intel compiler will be selected (in this case, the default is the version that you have selected)